### DISCUSSION TITLE
Interactive graph and list of backlinks like Obsidian Publish

### DISCUSSION THREAD

**Discussion by AlphaJack** [squidfunk/mkdocs-material#6655] [Ideas]
[Obsidian Publish](https://help.obsidian.md/Home) features an interactive graph between between pages that link to the current one. In addition, these notes are reported at the end of the page.

![image](https://github.com/squidfunk/mkdocs-material/assets/19962243/751fd772-d5f9-4da6-8c0c-3867aa390b53)

Can Material access a list of links to the current page to build such features?

**Comment by alexvoss** [comment]
Have a look at the [backlinks plugin](https://pypi.org/project/mkdocs-backlinks/). Perhaps it already does what you need? Not sure how much work it would be to add the graph but if the plugin does not already compile an index of the links (which I think it must do?) then it should be easy to do so using [hooks](https://www.mkdocs.org/user-guide/configuration/#hooks).

**Reply by AlphaJack** [reply]
Thank you, I did not know that plugin. I opened an [issue](https://github.com/danodic-dev/mkdocs-backlinks/issues/7) in its repo.
